bench/bench.py

```diff
@@ -415,6 +423,15 @@ def wheel(self, verbose=False):
 			f"{self.bench.python} -m pip install {quiet_flag} wheel", cwd=self.bench.name
 		)
 
+	@step(title="Checking if pkg-config is installed", success="pkg-config is present")
+	def check_pkg_config(self):
+		"""
+		pkg-config is required for building some python packages like libmysqlclient
+		"""
+		if shutil.which("pkg-config") is None:
+			raise Exception("pkg-config is not installed. Please install it before proceeding.\n"
```

Comment:
Maybe do this only for develop/v16+?

Reply:
Without this, this is a breaking change

Reply:
> Without this, this is a breaking change

Hmm fair. Will add in a check.

Reply:
We just pip install {app_name} for requirements setup, need to see how to get version in sanely.